plugins/out_kinesis_firehose/firehose_api.c

```diff
@@ -42,9 +42,12 @@
 #include <msgpack.h>
 #include <string.h>
 #include <stdio.h>
-#include <unistd.h>
 #include <stdlib.h>
 
+#ifndef FLB_SYSTEM_WINDOWS
+#include <unistd.h>
```

Comment:
Its not always safe to change include order. It may be better to put the ifndef inline

Reply:
Why is it not safe?

Reply:
For a file like `unistd.h` its probably just fine. Especially followed by `stdlib.h` since these are library includes I'm sure they have done the right thing.